lib/common/http2client.c

```diff
@@ -282,6 +283,26 @@ static void call_stream_callbacks_with_error(struct st_h2o_http2client_conn_t *c
     kh_foreach_value(conn->streams, stream, { call_callback_with_error(stream, errstr); });
 }
 
+static int extract_content_length(const h2o_headers_t *headers, size_t *content_length, const char **err_desc)
+{
+    *content_length = SIZE_MAX;
+    for (size_t i = 0; i < headers->size; ++i) {
+        if ((const h2o_token_t *)headers->entries[i].name == H2O_TOKEN_CONTENT_LENGTH) {
```

Comment:
```suggestion
        if (headers->entries[i].name == &H2O_TOKEN_CONTENT_LENGTH->buf) {
```
Purely editorial, but this is how we do elsewhere (when we need to compare only one token), and doing so avoids cast and therefore is less prone to errors.

Reply:
Good point, thank you, done in 303667b71.